### PR TITLE
[DE-513] Pandora products from shopify

### PR DESCRIPTION
**Pull request by fjunejo**

### CODE REVIEW

File: PandoraMarketplaceProducts/ci_requirements.txt [hunk 1]
```diff
@@ -0,0 +1,8 @@
+apache-beam[gcp]==2.24.*
```
Comment (AlexFragotsis) [inline PandoraMarketplaceProducts/ci_requirements.txt:1]: Do we need that ?

File: PandoraMarketplaceProducts/tools/SchemaValidator.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+from datetime import datetime
```
Comment (AlexFragotsis) [inline PandoraMarketplaceProducts/tools/SchemaValidator.py:1]: Is it possible to move that tools folder outside so all the templates can use the same?

**Review by AlexFragotsis — APPROVED** [review]
A couple minor comments only, looks good otherwize

File: PandoraMarketplaceProducts/ci_requirements.txt (hunk 1 above)
Comment (fjunejo) [reply PandoraMarketplaceProducts/ci_requirements.txt:1]: This is for the CircleCI image that builds the template. It needs the beam SDK to be able to compile the template.

File: PandoraMarketplaceProducts/tools/SchemaValidator.py (hunk 2 above)
Comment (fjunejo) [reply PandoraMarketplaceProducts/tools/SchemaValidator.py:1]: I was just thinking that! Will test it in a separate PR...

**Pull request merged**